<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainModule</class>
 <widget class="QWidget" name="MainModule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>901</width>
    <height>777</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>721</width>
     <height>661</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>이름</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>촬영날짜</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>파일크기</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>파일위치</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>환자 ID</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="runScanModule">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>90</y>
     <width>141</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>새로 스캔하기</string>
   </property>
  </widget>
  <widget class="QPushButton" name="runViewerModule">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>200</y>
     <width>141</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>이전스캔 불러오기</string>
   </property>
  </widget>
  <widget class="QPushButton" name="runCalibrationModule">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>310</y>
     <width>141</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>카메라 캘리브레이션</string>
   </property>
  </widget>
  <widget class="QPushButton" name="runExpModule">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>420</y>
     <width>141</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>데이터 검증실험</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>62</y>
     <width>721</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>이전 스캔 목록</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
